--- a/src/jpgraph_pie.php
+++ b/src/jpgraph_pie.php
@@ -567,7 +567,7 @@ class PiePlot {
 		$img->SetColor($this->color);
 	    else
 		$img->SetColor($slicecolor);	    
-	    $arccolor = $this->pie_border && $aaoption===0 ? $this->color : "";
+	    $arccolor = $this->pie_border && $aaoption===0 ? $slicecolor.':0.7' : "";
 
 	    // Calculate the x,y coordinates for the base of this slice taking
 	    // the exploded distance into account. Here we use the mid angle as the
